modules/ROOT/partials/integrations/webcomponent-quick-start.adoc

```diff
@@ -18,15 +18,19 @@ The `DOCTYPE` declaration is required for the editor to function correctly.
 
 . Add the following `meta` elements to the head of page:
 +
-```html+++<head>++++++<meta charset="utf-8">++++++</meta>++++++<meta name="viewport" content="width=device-width, initial-scale=1">++++++</meta>++++++</head>+++
-+
-```
+[source, html]
+----
+<head>
+  <meta charset="utf-8"></meta>
+  <meta name="viewport" content="width=device-width, initial-scale=1"></meta>
```

Comment:
```suggestion
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
```
This was something the conversion messed up. Anyways, again they are void elements so there should be no `</meta>`.

Reply:
done